lib/bootstrap.js

```diff
@@ -16,6 +16,7 @@ const extraAliases = {
     busters: 'buster',
     intsall: 'install',
     insall: 'install',
+    'status': 'ls',
```

Comment:
Didn't see your other comment before pushing those changes up, now unquoted in all its glory.